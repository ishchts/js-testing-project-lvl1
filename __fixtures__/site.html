<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="utf-8">
    <title>Курсы по программированию Хекслет</title>
    <link rel="stylesheet" media="all" href="https://cdn2.hexlet.io/assets/menu.css">
    <link rel="stylesheet" media="all" href="/application.css" />
  </head>
  <body>
    <img src="nodejs.png" alt="Иконка профессии Node.js-программист" />
    <h3>
      <a href="/professions/nodejs">Node.js-программист</a>
    </h3>
    <script src="https://js.stripe.com/v3/"></script>
    <script src="https://ru.hexlet.io/script.js"></script>
</body></html>