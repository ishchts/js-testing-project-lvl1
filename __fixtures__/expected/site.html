<!DOCTYPE html><html lang="ru"><head>
    <meta charset="utf-8">
    <title>Курсы по программированию Хекслет</title>
    <link rel="stylesheet" media="all" href="https://cdn2.hexlet.io/assets/menu.css">
    <link rel="stylesheet" media="all" href="ru-hexlet-io-courses_files/ru-hexlet-io-application.css">
  </head>
  <body>
    <img src="ru-hexlet-io-courses_files/ru-hexlet-io-nodejs.png" alt="Иконка профессии Node.js-программист">
    <h3>
      <a href="/professions/nodejs">Node.js-программист</a>
    </h3>
    <script src="https://js.stripe.com/v3/"></script>
    <script src="ru-hexlet-io-courses_files/ru-hexlet-io-script.js"></script>
</body></html>